Installing-Oppia-(Linux;-Python-3).md

```diff
@@ -0,0 +1,150 @@
+```
+pip install coverage configparser
+```
+## Running Oppia on a development server ##
+
+1. In a terminal, navigate to `oppia/` and run:
+
+  ```
+     python -m scripts.start
+  ```
+
+  The first time you run this script, it will take a while -- about 5 - 10 minutes when we last tested it in Sep 2020, though this depends on your Internet connection. (It might also hang after "Checking if pip is installed on the local machine" due to the grpcio build being slow -- just give it some time, and it should finish.) Subsequent runs should be much faster. The `start.py` script downloads and installs the required dependencies (such as Google App Engine) if they are not already present, and sets up a development server for you to play with. The development server logs are then output to this terminal, so you will not be able to enter further commands in it until you disconnect the server.
+
+  **Note**: **Please don't use `sudo` while installing.** It's not required, and using it may cause problems later. If you face permissions issues, ensure that you have the necessary permissions for the directory in which you're trying to set up Oppia. If you run into any other installation problems, please read [these notes](https://github.com/oppia/oppia/wiki/Issues-with-installation%3F).
+
+  **Note**: The script will create two folders that are siblings of the `oppia/` root directory: `oppia_tools` and `node_modules`. This is done so that these two folders will not be uploaded to App Engine when the application is deployed to the web.
+  
+  **Note**: If you run into errors while installing Oppia, please try running `python -m scripts.clean` and running `start.py` again.
+
+  **Note**: Oppia uses the npm tool to install some packages. This tool accesses both ~/tmp and ~/.npm, and has been known to occasionally encounter permissions issues with those directories. You may need to either delete these directories and all their contents (if they do not contain anything else that needs to be preserved), or change their permissions so that they are owned by you, which you can do by running
+
+  ```
+    sudo chown -R {{YOUR_USERNAME}} ~/tmp
+    sudo chown -R {{YOUR_USERNAME}} ~/.npm
+  ```
+
+  where `{{YOUR_USERNAME}}` should be replaced by your username.
+
+2. The `start.py` script will start a development server at http://localhost:8181. It should look something like this:
+
+  ![Image showing the default splash page.](https://res.cloudinary.com/dozmja9ir/image/upload/v1538254601/home_page.png)
+
+  You can also view the App Engine admin console at http://localhost:8000.
+
+  **Note:** There may be a few warnings that appear after running `start.py`. Don’t worry about these so long as you see the page above once you go to http://localhost:8181. The script should continue to run so long as the development server is on (you’ll see a lot of lines that start with “INFO”) and you’re able to navigate to the page. 
```

Comment:
is on --> is running

Reply:
This file change isn't actually part of this PR. I just forgot to push the latest `master` from `oppia/oppia` to the `master` of `U8NWXD/oppia-wiki`. I've fixed this now